serialization/src/main/scala/sbt/serialization/CustomPicklerUnpickler.scala

```diff
@@ -1,18 +1,23 @@
 package sbt.serialization
 
-import sbt.serialization.pickler.{
+// TODO - Why are type aliases not workign?
```

Comment:
Right. That's probably what's happening. It has to go traverse a tree before the package object is seen.  Yay.